client/src/app/site/polls/components/poll-list/poll-list.component.ts

```diff
@@ -49,4 +51,12 @@ export class PollListComponent extends BaseListViewComponent<ViewBasePoll> {
     ) {
         super(title, translate, snackbar, storage);
     }
+
+    public getContentObjectTitle(poll: ViewBasePoll): string {
```

Comment:
Hmm, please make a `getContentObject` required in the `BaseViewPoll`, so `MotionViewPoll` and `AssignmentViewPoll` must implement this. The Returntype may be sometzhing like `BaseViewModel`. Than you can do `poll.getContentObject().getListTitle()` in the template. (See Open-Closed-Principle)